example/resolver.js

```diff
@@ -78,24 +78,22 @@ async function main() {
 
   console.log('Building DIDs list...');
 
+  const dockDID = await createDockDID();
```

Comment:
had to make a side change here, example was hanging in the CI process. some dids were duplicates and/or not resolving properly, causing a hang but not an error. something in universal resolver side